src/vm/amd64/asmconstants.h

```diff
@@ -663,7 +663,9 @@ ASMCONSTANTS_C_ASSERT(OFFSETOF__TransitionBlock__m_argumentRegisters == offsetof
 
 #undef ASMCONSTANTS_RUNTIME_ASSERT
 #undef ASMCONSTANTS_C_ASSERT
+#ifndef UNIX_AMD64_ABI
 #undef DBG_FRE
```

Comment:
DBG_FRE is used by OFFSETOF__MethodTable__m_ElementType, which is used by JIT_Stelem_Ref. If this micro is undef here, the build process complains "unknown token in expression".